src/Windows/dcap_provider.vcxproj

```diff
@@ -47,7 +47,7 @@
       <AdditionalIncludeDirectories>$(PkgCurl)\inc</AdditionalIncludeDirectories>
       <ControlFlowGuard>Guard</ControlFlowGuard>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
-      <AdditionalIncludeDirectories>$(MSBuildProjectDirectory);$(MSBuildProjectDirectory)\ext;$(NugetMachineInstallRoot)\Json11.1.0.1\build</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(MSBuildProjectDirectory);$(MSBuildProjectDirectory)\..;$(MSBuildProjectDirectory)\ext;$(NugetMachineInstallRoot)\Json11.1.0.1\build</AdditionalIncludeDirectories>

```

Comment:
Just out of curiosity, what new header are you pulling from the parent of your project dir? That seems like a bad idea.

Reply:
I believe this was added so that the windows builds will be able to refer to the local_cache.h residing at the top level src directory.